pkg/kubelet/kubelet.go

```diff
@@ -1469,7 +1469,10 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 
 	if kl.kubeClient != nil {
 		// Start syncing node status immediately, this may set up things the runtime needs to run.
-		go wait.Until(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, wait.NeverStop)
+		// Introduce some small jittering to ensure that over time the requests won't start
+		// cumulating at approximately the same time from the set of nodes due to priority and
+		// fairness effect.
+		go wait.JitterUntil(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, 0.1, true, wait.NeverStop)
```

Comment:
Why are we using a bigger interval here than in #101652? (0.1 vs. 0.04)

Reply:
I think that 0.04 is fairly small - but I don't have strong opinion here.
I switched back to 0.04 - we can always change that later...